x-pack/plugin/rollup/src/main/java/org/elasticsearch/xpack/rollup/action/TransportRollupSearchAction.java

```diff
@@ -288,7 +288,7 @@ static QueryBuilder rewriteQuery(QueryBuilder builder, Set<RollupJobCaps> jobCap
             String fieldName = range.fieldName();
             // Many range queries don't include the timezone because the default is UTC, but the query
             // builder will return null so we need to set it here
-            String timeZone = range.timeZone() == null ? DateTimeZone.UTC.toString() : range.timeZone();
+            String timeZone = range.timeZone() == null ? ZoneOffset.UTC.toString() : range.timeZone();
```

Comment:
same as above

Reply:
That is read from `RangeQueryBuilder` 
does not seem to be using DateUtils
` timeZone = in.readOptionalZoneId();`
do you think that should be fixed in server?